### PR TITLE
Remove .pyc files when developer venv is (re)created

### LINKED ISSUE #1342: Persistence of .pyc files causes periodic local tox failures
I just wasted a bunch of time chasing down another instance of this. We should probably rm them in the `dev/venv.sh` script, at the very least.

### PR DESCRIPTION
**Pull request by pde**
Fixes #1342, at least for users who recreate their venv/ dir

### CODE REVIEW

**Comment by kuba** [comment]
note: users shouldn't recreate their venv; why would they rerun the script?

File: bootstrap/dev/venv.sh [hunk 1]
```diff
@@ -11,3 +11,5 @@ export VENV_ARGS="--python python2"
   -e letsencrypt-nginx \
   -e letshelp-letsencrypt \
   -e letsencrypt-compatibility-test
+
+find . -iname '*.pyc' -exec rm '{}' ';'
```
Comment (devnsec-com) [inline bootstrap/dev/venv.sh]: What if someone (accidentally) run `/path/to/bootstrap/dev/venv.sh` from a directory which is not a part of LE but contains `*.pyc` files?

Comment (pde) [inline bootstrap/dev/venv.sh]: The previous command would error, causing the script to exit due to -e.  But I can double-mitigate this.

Comment (devnsec-com) [inline bootstrap/dev/venv.sh]: I think `-e` in the shellbang is not really safe enough for this. For an example, if the user types in `bash /path/to/bootstrap/dev/venv.sh`

**Comment by pde** [comment]
Postponing until after **1.0 for launch** is done.

**Pull request closed without merging**